code/modules/clothing/glasses/_glasses.dm

```diff
@@ -295,17 +314,20 @@
 	desc = "Protects the eyes from bright flashes; approved by the mad scientist association."
 	icon_state = "welding-g"
 	inhand_icon_state = "welding-g"
-	actions_types = list(/datum/action/item_action/toggle)
```

Comment:
this is a pretty huge nerf to welding masks, and exactly what I feared would come from this PR.

I normally keep my welding mask's/welding gas mask's action button close to the center of my screen, so that I can flick it on and off when relevant. This PR makes it so that I have to drag my mouse all the way over to the left side of my screen to do that. I know that doesn't sound like much, but for me, it's similar to the difference between having and not having quickdraw hotkeys, and is part of what makes playing a moth bearable for me in this flash-infested world.

Reply:
Pretty huge nerf to trying to counter a flash as a moth? Just look away, jesus fuck. They do nothing if you're facing the opposite direction

Reply:
> Pretty huge nerf to trying to counter a flash as a moth? Just look away, jesus fuck. They do nothing if you're facing the opposite direction

not all of us play pacifists, hugbug

also, I use lit welders to attack sometimes, which requires flipping down my visor just before I attack to not nuke my eyes, and then flipping it up again after my target leaves melee range so that I can continue to track them down

Reply:
https://streamable.com/s6v3w8
Why do I have to teach you about combat mechanics, despite being a pacifist? 
Aren't you supposed to be the sweaty powergamer in this relationship?

Reply:
> https://streamable.com/s6v3w8
> Why do I have to teach you about combat mechanics, despite being a pacifist?
> Aren't you supposed to be the sweaty powergamer in this relationship?

... huh. The more ya know.